src/DB_Command.php

```diff
@@ -1457,13 +1482,46 @@ public function search( $args, $assoc_args ) {
 							$match_count += $match_cnt;
 							$col_val      = implode( ' [...] ', $bits );
 
-							WP_CLI::log( $matches_only ? $col_val : ( $one_line ? "{$table_column_val}:{$pk_val}{$col_val}" : "{$pk_val}{$col_val}" ) );
+							if ( $format ) {
+								$search_results[] = [
+									'table'  => $table,
+									'column' => $column,
+									'key'    => $primary_key,
+									'ID'     => $result->$primary_key,
```

Comment:
Should this be `'value'` instead of `'ID'`? It seems a little bit weird to call this `'ID'` when the primary key column could be called something else.